### ISSUE TITLE
Episode 6:  Working with columns and sorting

### ISSUE THREAD

**Issue by fionaglasgow** [LibraryCarpentry/lc-open-refine#94]
Kia ora, 

As part of the checkout process I propose a minor change to Chapter 6:  Working with columns and sorting.

I suggest a minor change of wording relating to re-order vs reorder:

> You can re-order the columns by clicking the drop-down menu at the top of the first column (labelled ‘All’), and choosing Edit columns->Re-order / remove columns …
> 
> You can then drag and drop column names to re-order the columns, or remove columns completely if they are not required.

to be replaced by:

> You can reorder the columns by clicking the drop-down menu at the top of the first column (labelled ‘All’), and choosing Edit columns->Re-order / remove columns …
> 
> You can then drag and drop column names to reorder the columns, or remove columns completely if they are not required.

Motivation for suggested change:

There are two variations on "reorder" in use on this page, which, while minor, is inconsistent. 

It might be worth having a discussion to see which way it goes, ie, the "reorders" could be changed to "re-orders", given that that is how it is spelled in OpenRefine, even though I don't believe that is technically grammatically correct. The remainder of the lesson uses reorder, for what it's worth.

Fiona Glasgow
Research Support Librarian
University of Otago

**Comment by fionaglasgow** [comment]
commit --amend

**Comment by ostephens** [comment]
I agree this should be consistent throughout and that reorder (non-hyphenated) is the correct, or at least more common, form.

Apart from where the text is specifically reproducing the terminology from OpenRefine (eg directing the user to a menu) I think we should use ‘reorder’

A PR to this effect would be welcome
